recipes-bsp/flashrom/flashrom_git.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "flashrom"
+DESCRIPTION = "flashrom is a utility for identifying, reading, writing, verifying and erasing flash chips"
+LICENSE = "CLOSED"
+HOMEPAGE = "https://github.com/pcengines/flashrom"
+
+SRC_URI = "git://github.com/pcengines/flashrom.git;branch=apu_support "
+SRCREV = "092b1c28bf31dab170d2fc0dc09b6bee3fa68bfa"
+
+DEPENDS = "pciutils libusb libusb-compat"
+
+PV = "1.0.1-pcengines+${SRCPV}"
```

Comment:
@Asiderr Where did you set which flashrom  version shall be used?

Reply:
@macpijan I set the `PREFERRED_PROVIDER_pcengines/bsp = "flashrom"` in the apu2-diagnostic-image.bb file.

Reply:
This is really weird config. Use `PREFERRED_Version_flashrom = "1.0.1-pcengines%"` and place it in distro config file (not in image recipe).

Reply:
@macpijan Changed in the last commit.